Library/ViewModels/SettingsViewModel.swift

```diff
@@ -675,22 +690,23 @@ private enum Notification {
     case .creatorTips:                              return "Creator tips"
     case .follower, .mobileFollower:                return "New followers"
     case .friendActivity, .mobileFriendActivity:    return "Friend backs a project"
-    case .messages, .mobileMessages:                 return "New messages"
+    case .messages, .mobileMessages:                return "New messages"
     case .postLikes, .mobilePostLikes:              return "New likes"
     case .updates, .mobileUpdates:                  return "Project updates"
     }
   }
 }
 
 private enum Privacy {
-
   case following
   case recommendations
+  case showPublicProfile
 
   fileprivate var trackingString: String {
     switch self {
     case .following: return Strings.Following()
     case .recommendations: return Strings.Recommendations()
+    case .showPublicProfile: return "Show public profile"
```

Comment:
Something that's handy to do here while waiting for a translation is to write it out as `localizedString(key: "Show_public_profile", defaultValue: "Show public profile")`, that way once the translation comes through you can just do a search on the project for instances of `localizedString` to be sure you've caught them all.

Reply:
Ok so I wasn't sure about this one - do we actually want the localized string since its a tracking string? 🤔I thought we might want the english version always...

Reply:
Oh sure, I just noticed that the others were localized, not 💯sure about the use case here.

Reply:
You know what I just double checked with Nneka and we don't need to track this anyway. I've removed this tracking string and added a default instead.